ReactiveSwift/Scheduler.swift

```diff
@@ -144,7 +144,7 @@ public final class QueueScheduler: DateSchedulerProtocol {
 		return Date()
 	}
 
-	internal let queue: DispatchQueue
+	public let queue: DispatchQueue
```

Comment:
It's used [here](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/e053ec2a08d4703b0150a84bae9f826fe3579433/ReactiveCocoa/Swift/ObjectiveCBridging.swift#L88), so I made it `public`. 😕 

I have mixed feelings about it, but it doesn't seem terrible to expose it. I'm open to alternatives!